daprdocs/content/en/operations/resiliency/policies.md

```diff
@@ -91,3 +93,40 @@ spec:
+
+> Note: Although you can override default values with more robust retries, you cannot override with lesser values than the provided default value, or completely remove default retries. This prevents unexpected downtime. 
+
+Below is a table that describes Dapr's default retries and the policy keywords to override them: 
+
+| Capability             | Override Keyword                 | Default Retry Behavior                                                                                                                                                               | Description                                                                                                                           |
+| ------------------     | -------------------------        | ------------------------------                                                                                                                                                       | -----------------------------------------------------------------------------------------------------------                           |
+| Service Invocation     | DaprBuiltInServiceRetries        | Per call retries are performed with a backoff interval of 1 second, up to a threshold of 3 times.                                                                                     | Sidecar-to-sidecar requests (a service invocation method call) that fail and result in a gRPC code `Unavailable` or `Unauthenticated`   | 
+| Actors                 | DaprBuiltInActorRetries          | Per call retries are performed with a backoff interval of 1 second, up to a threshold of 3 times.                                                                                     | Sidecar-to-sidecar requests (an actor method call) that fail and result in a gRPC code `Unavailable` or `Unauthenticated`              |
+| Actor Reminders        | DaprBuiltInActorReminderRetries  | Per call retries are performed with an exponential backoff with an initial interval of 500ms, up to a maximum of 60s for a duration of 15mins                                 | Requests that fail to persist an actor reminder to a state store                                                                      | 
+| Initialization Retries | DaprBuiltInInitializationRetries | Per call retries are performed 3 times with an exponential backoff, an initial interval of 500ms and for a duration of 10s                                                       | Failures when making a request to an application to retrieve a given spec. For example, failure to retrieve a subscription, component or resiliency specification       | 
+
+
+Example of overriding default retries for Service Invocation:
```

Comment:
```suggestion
Below is an example of a Resiliency spec that includes overriding default retries for all service invocation requests by using the reserved, named keyword 'DaprBuiltInServiceRetries'. Also defined is a custom retry policy called 'retryForever' that is only applied to the appB target. 

appB will use the 'retryForever' retry policy, while all other service invocations failures will default to using the 'DaprBuiltInServiceRetries' override policy.
```

Reply:
@msfussell, does this help clarify what the resiliency spec example demonstrates?